guides/release/templates/actions.md

```diff
@@ -21,22 +21,30 @@ export default class Post extends Component {
 }
 ```
 
-You can then add this action to an element using the
-[`{{action}}`](https://api.emberjs.com/ember/release/classes/Ember.Templates.helpers/methods/action?anchor=action)
+`@action` binds the `this` of `toggleBody()` to the instance of the class, allowing access the component's other properties when invoked. Without the decorator, `this.isShowingBody` in `toggleBody()` is undefined. For more information on `this` [Understanding Javascript Function Invocation and this](https://yehudakatz.com/2011/08/11/understanding-javascript-function-invocation-and-this/)
```

Comment:
Thanks @NullVoxPopuli and @locks for talking this through with me. I think this hits the key terms (`binds`, `class`) while also, with the second sentence, explaining in a more quotidian way why `@action` is needed!